manager-dashboard/app/views/NewTutorial/utils.ts

```diff
@@ -651,13 +693,20 @@ export const tutorialFormSchema: TutorialFormSchema = {
                                         return undefined;
                                     }
 
+                                    const hasDuplicateSubOptionValue = checkDuplicate(
+                                        sub.map((val) => val.value).filter(isDefined),
+                                    );
```

Comment:
We do not need this validation as this is covered from another validation above.